src/rpm/headers/header.rs

```diff
@@ -538,7 +538,7 @@ impl Header<IndexTag> {
                         group: group.to_owned(),
                     },
                     mode: mode.into(),
-                    modified_at: utc.timestamp(mtime as i64, 0u32),
+                    modified_at: utc.timestamp_opt(mtime as i64, 0u32).unwrap(),
```

Comment:
e.g. why is this better

Reply:
`timestamp` got deprecated with the latest `chrono` release

Reply:
It will also make clippy pass again in other PRs once they are rebased.

Reply:
yeah